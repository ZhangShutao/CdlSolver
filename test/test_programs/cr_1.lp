s.
p | q :- not t, not &c{not apply(2)}.