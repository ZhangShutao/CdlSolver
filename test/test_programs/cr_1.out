default model: 1
x:s; y:s;
Satisfiable